analyzer/config/yaml/yaml_test.go

```diff
@@ -0,0 +1,108 @@
+package yaml
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/open-policy-agent/conftest/parser/yaml"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/aquasecurity/fanal/analyzer"
+	"github.com/aquasecurity/fanal/analyzer/config"
+	"github.com/aquasecurity/fanal/types"
+)
+
+func Test_yamlConfigAnalyzer_Analyze(t *testing.T) {
```

Comment:
If there is a possibility of changing the yaml library in the future, I think it is better to test the required implementation.

e.g.
```
hoge: &hoge
  line: single line
fuga:
  <<: *hoge
  comment: |
    multi
    line
```

Reply:
Add test commit.